lib/util/cache/package/decorator.ts

```diff
@@ -42,6 +44,15 @@ interface CacheParameters {
   ttlMinutes?: number;
 }
 
+function getClassName(instance: unknown): string | undefined {
+  // istanbul ignore if: should never happen
+  if (!is.object(instance)) {
+    return undefined;
+  }
+
+  return instance.constructor.name;
```

Comment:
would it make sense to do `kebab-case` here? would look much better 🤔

Reply:
Is it worth [introducing and maintaining the new code](https://github.com/renovatebot/renovate/pull/31060/files#diff-3744a7eafe7880efbc56d2f472cf9d5588ddeb724e8d233d10d8f78430a49bdcR3-R44), given that mostly we don't work directly with it in the underlying storage level?

Reply:
don't we already have lodash in our prod dependency chain which already has such a function?

Reply:
I still think it's better to just use class name unchanged. As far as we don't want to reuse older cache keys, the less name mangling is better.

Reply:
For example, if the underlying string converter slightly changes its behavior (e.g. for abbreviations), the dependency update will cause unanticipated cache reset. And we may spend time investigating load spike, or something like that.